dice/.htaccess

```diff
@@ -0,0 +1,3 @@
+<FilesMatch "\.(key|dat)$">
+  deny from all
+</FilesMatch>
```

Comment:
I'm not sure if this file is going to affect anything since we're using nginx and not apache.